plugins/server/com.ucasoft/group.ktor.yaml

```diff
@@ -0,0 +1,3 @@
+name: UCASoft
+url: https://ucasoft.com
```

Comment:
You might want to introduce a logo for your org in this directory to differentiate the plugins in the generator application as well.  See `plugins/group/org.postresql` for example.

Reply:
Yeah, to be honest, I don't have any, yet.
Is it mandatory?

Reply:
No problem, it's totally optional!

Reply:
I built one via AI, but I'm not sure yet. So, let's keep it without logo for a while.